FV3/wrapper/tests/test_all_mpi_requiring.py

```diff
@@ -28,14 +28,17 @@ def test_diagnostics(self):
     def test_tracer_metadata(self):
         run_unittest_script("test_tracer_metadata.py")
 
-    def test_get_time_julian(self):
-        run_unittest_script("test_get_time.py", "julian")
+    def test_get_time_calendar_julian(self):
+        run_unittest_script("test_get_time_calendar.py", "julian")
 
-    def test_get_time_thirty_day(self):
-        run_unittest_script("test_get_time.py", "thirty_day")
+    def test_get_time_calendar_thirty_day(self):
```

Comment:
nit: These are two test cases of one function, I would suggest putting them both in a `test_get_time.py` file with no other changes (the names of the classes/functions/methods are fine).

Reply:
I think this will end up running the time getter tests repeatedly; `test_get_time_calendar` is invoked for each calendar argument, which is why I put the time getter test in a different file. I don't think it will be very slow to repeat this test so maybe that's fine.

Reply:
It looks like both the original "get_time" tests and the "advances" tests are calendar-dependent, so this is probably good coverage?